core/src/main/java/io/neow3j/contract/Hash160.java

```diff
@@ -129,18 +145,15 @@ public static Hash160 fromAddress(String address) {
      * @return the script hash.
      */
     public static Hash160 fromScript(byte[] script) {
-        // There is no need to reverse the hash. The hashing method returns the script hash in
-        // little-endian format.
-        return new Hash160(Hash.sha256AndThenRipemd160(script));
+        return new Hash160(reverseArray(sha256AndThenRipemd160(script)));
```

Comment:
I don't trust this anymore. As far as i can tell the `sha256AndThenRipemd160` method is returning big-endian, but in the javadoc of that method i wrote it returns the hash in little-endian ordering. I want to run some tests on this branch.

Reply:
Seems fine, according to my tests.